cocos/scripting/js-bindings/manual/js_manual_conversions.cpp

```diff
@@ -243,8 +248,13 @@ bool jsval_to_uint( JSContext *cx, JS::HandleValue vp, unsigned int *ret )
     // Since this is called to cast uint64 to uint32,
     // it is needed to initialize the value to 0 first
 #ifdef __LP64__
-    long *tmp = (long*)ret;
-    *tmp = 0;
+    // only unsigned int size is 8Bit, same with the long, it's need to reset the data.
```

Comment:
When `unsigned int` size is 8 Bit (same as `long`), the following operation is needed